rt_gene/scripts/estimate_blink.py

```diff
@@ -80,10 +80,9 @@ def callback(self, msg):
             '\033[2K\033[1;32mTime now: {:.2f} message color: {:.2f} latency: {:.2f}s for {} subject(s) {:.0f}Hz\033[0m'.format(
                 (_now.to_sec()), timestamp.to_sec(), np.mean(self._latency_deque), len(subjects), np.mean(self._freq_deque)), end="\r")
 
-    def publish_msg(self, timestamp, subjects, probabilities):
+    def publish_msg(self, header, subjects, probabilities):
         blink_msg_list = MSG_BlinkList()
-        blink_msg_list.header.stamp = timestamp
-        blink_msg_list.header.frame_id = '0'
+        blink_msg_list.header.stamp = header
```

Comment:
Should this be `blink_msg_list.header = header`?

Reply:
You're quite right...
One day my tests will show up these errors, but today is not that day...